tests/css_bom/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`bom.css 1`] = `
+﻿
```

Comment:
There _is_ a BOM here but it doesn't show up in the diff

![image](https://user-images.githubusercontent.com/1297597/27761121-6139ca50-5e99-11e7-8cc2-e12febc23f09.png)